modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/ReplicaListenerHelper.java

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
You are going to remove

- ReplicaListenerHelper
- ReplicaTxFinishHelper
- RaftTxFinishHelper

right?